views/js/runner/proxy/cache/actionStore.js

```diff
@@ -81,6 +81,22 @@ define([
                         });
                     });
                 });
+            },
+
+            /**
+             * update action parameters
+             * @param params
```

Comment:
Please indicate the type of params, `{Object}` it may looks obvious but it's the rule